### DISCUSSION TITLE
Add a README.md file outside of _internal

### DISCUSSION THREAD

**Discussion by Equalizer5118** [pyinstaller/pyinstaller#8869] [Help] [answered]
Hey there, i've just started using pyinstaller and I wanted to automatically add a readme and a version file into the same folder as the EXE file, so users can read how to use the program without having to go back to the repo my program is being stored in. Ive added the files using the `datas=` parameter in my `.spec` file, and they are being placed in the `_internal` folder. While I understand this is expected behaviour, is there a way or a different parameter that would put these files outside of the `_internal` folder, being placed along side my EXE? 
Thanks for the help.

**Comment by rokm** [comment]
If you want to place a file next to the executable, you'll have to copy it yourself. `datas=` is strictly for the application contents, which goes into the application's top-level directory (i.e., the `_internal` directory).

**Reply by Equalizer5118** [reply]
Could you edit the docs to reflect that? The docs use a readme file as an example, hence why I got confused. https://pyinstaller.org/en/stable/spec-files.html#adding-data-files

**Reply by rokm** [reply]
That's a good point, thanks!

That readme file example was a poor choice as introductory example even before we moved top-level application directory of one-dir applications to the `_internal` sub-directory, since it was not applicable to onefile builds.

**Comment by bwoodsend** [comment, marked as answer]
If it's going outside the `_internal` directory then you can just `shutil.copy()` it at the end of your `.spec` file.

**Reply by Equalizer5118** [reply]
thanks, forgot that the spec file is a python script!
